application/migrations/20240516002600_add_keywords_instances_field.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+defined('BASEPATH') OR exit('No direct script access allowed');
+
+class Migration_Add_keywords_instances_field extends CI_Migration {
+
+        public function up()
+        {
+		$fields = array(
+        		'instances' => array('type' => 'INT',
```

Comment:
This is a count, right? Or put differently, how many instances of this keyword is applied to projects? I hate to bikeshed on things like this, but because this will be in our database schema, getting the name right the first time around is important. I would strongly prefer we use `count` rather than `instances`. There's precedent as well: we have a `section_status_count` table.

Reply:
Agreed